hasher-matcher-actioner/hmalib/indexers/tests/test_lcc_indexer.py

```diff
@@ -26,8 +28,13 @@ def test(self):
             )
             tbi.force_flush()
             test_class = LCCIndexer()
-            test_index = test_class.build_index_from_last_24h(
-                tbi.type, tbi.storage_path, tbi.bucket_width
-            )
+            with metrics.timer(metrics.names.lcc.build_index):
```

Comment:
I don't think you should be checking in changes to this file as part of the PR. We only wanted to generate some data so that your dashboard was not empty.